backend/src/schema/resolvers/notifications.spec.js

```diff
@@ -1,26 +1,34 @@
 import { GraphQLClient } from 'graphql-request'
 import Factory from '../../seed/factories'
 import { host, login, gql } from '../../jest/helpers'
+import { neode } from '../../bootstrap/neo4j'
 
-const factory = Factory()
 let client
+const factory = Factory()
+const instance = neode()
 const userParams = {
   id: 'you',
   email: 'test@example.org',
   password: '1234',
 }
+const categoryIds = ['cat9']
 
 beforeEach(async () => {
   await factory.create('User', userParams)
+  await instance.create('Category', {
+    id: 'cat9',
+    name: 'Democracy & Politics',
+    icon: 'university',
+  })
 })
 
 afterEach(async () => {
   await factory.cleanDatabase()
 })
 
-describe('query for notification', () => {
+describe('Notification', () => {
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-21T10:00:51Z" title="Wednesday, August 21st 2019, 12:00:51 pm +02:00">Aug 21, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,26 +1,34 @@
 import { GraphQLClient } from 'graphql-request'
 import Factory from '../../seed/factories'
 import { host, login, gql } from '../../jest/helpers'
+import { neode } from '../../bootstrap/neo4j'
 
-const factory = Factory()
 let client
+const factory = Factory()
+const instance = neode()
 const userParams = {
   id: 'you',
   email: 'test@example.org',
   password: '1234',
 }
+const categoryIds = ['cat9']
 
 beforeEach(async () => {
   await factory.create('User', userParams)
+  await instance.create('Category', {
+    id: 'cat9',
+    name: 'Democracy & Politics',
+    icon: 'university',
+  })
 })
 
 afterEach(async () => {
   await factory.cleanDatabase()
 })
 
-describe('query for notification', () => {
+describe('Notification', () => {
```


this was an unintended change @Tirokk when dealing with a merge conflict, I can revert it, if you want